iocage_cli/update.py

```diff
@@ -29,11 +29,27 @@
 
 
 @click.command(
-    name="update",
-    help="Run freebsd-update to update a specified "
-    "jail to the latest patch level.")
-@click.argument("jail", required=True)
-def cli(jail):
+    name='update',
+    help='Run freebsd-update to update a specified '
+    'jail to the latest patch level.'
+)
+@click.option(
+    '--pkgs/--nopkgs', default=False,
+    help='Decide whether or not to update the pkg repositories and '
+         'all installed packages in jail.'
+)
+@click.option(
+    '--plugins/--noplugins', '-P/-NP', default=True,
```

Comment:
This doesn't make sense, they're calling update, whatever the type is should always be updated.

Reply:
There's a usecase, say latest plugin update version has bugs. So they would only like to update freebsd jail. In that case they are able to do so. We default to `True` so this is executed in most cases.

Reply:
That isn't necessary, the pkg flag would control that. The jail should always be updated. It's why they're calling it :P